src/Microsoft.AspNetCore.Authentication.MicrosoftAccount/MicrosoftAccountOptions.cs

```diff
@@ -23,6 +25,12 @@ public MicrosoftAccountOptions()
             TokenEndpoint = MicrosoftAccountDefaults.TokenEndpoint;
             UserInformationEndpoint = MicrosoftAccountDefaults.UserInformationEndpoint;
             Scope.Add("https://graph.microsoft.com/user.read");
+
+            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "id");
```

Comment:
Now that mapper is gone from everywhere else, maybe Import/Copy is better, since map is kinda vague what we are mapping.

CopyJsonKey
ImportJsonKey